### ISSUE TITLE
Assert error in smp_find_so(), storage/storage_persistent_silo.c

### ISSUE THREAD

**Issue by bsdphk** [varnishcache/varnish-cache#1790]
Old ticket imported from Trac
See archived copy here: https://varnish-cache.org/trac/ticket/1790

**Comment by lkarsten** [comment]
For easier access, here is the panic output in the trac link above:
```
Assert error in smp_find_so(), storage/storage_persistent_silo.c line 332:
  Condition(priv2 <= sg->p.lobjlist) not true.
thread = (cache-worker)
version = varnish-4.1.0-beta1 revision 0776128
ident = Linux,2.6.32-504.12.2.el6.x86_64,x86_64,-jnone,-sdeprecated_persistent,-smalloc,-hcritbit,epoll
Backtrace:
  0x43be32: pan_backtrace+0x1d
  0x43c228: pan_ic+0x2a6
  0x47a553: smp_find_so+0x6e
  0x47a843: smp_oc_getobj+0x17d
  0x43804d: obj_getobj+0x11c
  0x439e4f: ObjSetattr+0x134
  0x43a56c: ObjSetU32+0x38
  0x426145: vbf_beresp2obj+0x284
  0x428048: vbf_stp_fetch+0x636
  0x429642: vbf_fetch_thread+0x392
busyobj = 0x7f80d8090970 {
  ws = 0x7f80d8090a30 {
    id = "bo",
    {s,f,r,e} = {0x7f80d80928e8,+384,(nil),+57440},
  },
  refcnt = 2,
  retries = 0, failed = 0, state = 1,
  flags = {do_stream, is_gunzip},
  http_conn = 0x7f80d8092960 {
    fd = 16,
    doclose = NULL,
    ws = 0x7f80d8090a30,
    {rxbuf_b, rxbuf_e} = {0x7f80d8092a10, 0x7f80d8092a3c},
    {pipeline_b, pipeline_e} = {(nil), (nil)},
    content_length = -1,
    body_status = chunked,
    first_byte_timeout = 60.000000,
    between_bytes_timeout = 60.000000,
  },
  director_req = 0x2581c58 {
    vcl_name = s1,
    type = backend {
      display_name = vcl1.s1,
      ipv4 = 127.0.0.1,
      port = 41959,
      hosthdr = 127.0.0.1,
      health=healthy, admin_health=probe, changed=1441956434.3,
      n_conn = 1,
    },
  },
  director_resp = director_req,
  http[bereq] = 0x7f80d8090ff8 {
    ws[bo] = 0x7f80d8090a30,
    hdrs {
      "GET",
      "/1",
      "HTTP/1.1",
      "X-Forwarded-For: 127.0.0.1",
      "Accept-Encoding: gzip",
      "X-Varnish: 1002",
      "Host: 127.0.0.1",
    },
  },
  http[beresp] = 0x7f80d8091470 {
    ws[bo] = 0x7f80d8090a30,
    hdrs {
      "HTTP/1.1",
      "200",
      "OK",
      "Transfer-encoding: chunked",
      "Date: Fri, 11 Sep 2015 07:27:14 GMT",
    },
  },
  objcore[fetch] = 0x7f80c0000950 {
    refcnt = 2,
    flags = 0x2,
    objhead = 0x7f80c0000a00,
    stevedore = 0x2573130 (deprecated_persistent s0),
  },
  vcl = {
    temp = warm
    srcname = {
      "input",
      "Builtin",
    },
  },
},
```

**Comment by nigoroll** [comment]
I'm tempted to close this one because we haven't seen it for a long time with more recent code, IIRC
In case anyone is watching this ticket, please yell. Otherwise it will get closed after 5.1 release (March 15th)

**Comment by Dridi** [comment]
ETIMEDOUT
